docs/docs/developer/frontend/work-with-figma.mdx

```diff
@@ -47,6 +47,10 @@ For more comprehensive details and guidance on learning the Figma platform, you
 
 Select an element, hold `Option` key (Mac) or `Alt` key (Windows), then hover over another element to see the distance between them.
 
+### Figma extension for VSCode
+
+Figma for VS Code lets you navigate and inspect design files, collaborate with designers, track changes, and speed up implementation - all without leaving your text editor.
```

Comment:
Could you add a link to the extension + say a word about the fact it's part of the recommended extensions? (link to the file in github)